java/sage/ZRoot.java

```diff
@@ -35,19 +35,16 @@ public ZRoot(UIManager inUIMgr)
     super();
     uiMgr = inUIMgr;
     vf = uiMgr.getVideoFrame();
-    if (!Sage.EMBEDDED)
-    {
-      addComponentListener(this);
-      addMouseListener(this);
-      addMouseMotionListener(this);
-      addHierarchyListener(this);
-      vf.addMouseListeners(this);
-      vf.addMouseMotionListeners(this);
-    }
+    addComponentListener(this);
+    addMouseListener(this);
+    addMouseMotionListener(this);
+    addHierarchyListener(this);
+    vf.addMouseListeners(this);
+    vf.addMouseMotionListeners(this);
 
     alive = true;
 
-    if (Sage.EMBEDDED || uiMgr.getBoolean("ui/remote_render", false))
+    if (uiMgr.getBoolean("ui/remote_render", false))
```

Comment:
@Narflex - Something interesting is happening here, and I'm not sure why.  On Linux, I'm seeing that I'm getting a constant error int he log.

```
Sat 7/1 7:16:37.331 [FinalRender-SAGETV_PROCESS_LOCAL_UI@12465613] About to intialize DX9 renderer(1) in standard mode with size 3840x1200
Sat 7/1 7:16:37.331 [FinalRender-SAGETV_PROCESS_LOCAL_UI@12465613] Final Render had an error:java.lang.UnsatisfiedLinkError: sage.DirectX9SageRenderer.initDX9SageRenderer0(IIJ)Z
Sat 7/1 7:16:37.331 [FinalRender-SAGETV_PROCESS_LOCAL_UI@12465613] java.lang.UnsatisfiedLinkError: sage.DirectX9SageRenderer.initDX9SageRenderer0(IIJ)Z
Sat 7/1 7:16:37.331 [FinalRender-SAGETV_PROCESS_LOCAL_UI@12465613] sage.DirectX9SageRenderer.initDX9SageRenderer0(Native Method)
Sat 7/1 7:16:37.331 [FinalRender-SAGETV_PROCESS_LOCAL_UI@12465613] sage.DirectX9SageRenderer.allocateBuffers(DirectX9SageRenderer.java:91)
Sat 7/1 7:16:37.331 [FinalRender-SAGETV_PROCESS_LOCAL_UI@12465613] sage.ZRoot.executeFinalRenderCycle(ZRoot.java:472)
Sat 7/1 7:16:37.331 [FinalRender-SAGETV_PROCESS_LOCAL_UI@12465613] sage.ZRoot.executeFinalRenderCycle(ZRoot.java:446)
Sat 7/1 7:16:37.331 [FinalRender-SAGETV_PROCESS_LOCAL_UI@12465613] sage.ZRoot$1.run(ZRoot.java:357)
```
I can see that in this file, it tries to instantiate the Direct X Render in 3 places... And the only thing that I can see causing that is the renderType is set to NATIVE3DRENDER, and in which case, it wil use DirectX renderer.  I tested this on my production server (where it doesn't have these changes), and I'm not getting this error in the logs, but I do on my local system.

I'm connecting via the Linux placeshifter.

You have any thoughts on where I can track this down?